architectures/BDR-Always-ON/commands/upgrade.yml

```diff
@@ -25,6 +25,25 @@
 # we would prefer the downtime to be predictable rather than dependent
 # on the whims of the network.)
 
+- name: Check that the cluster can be upgraded
```

Comment:
```suggestion
- name: Record current BDR major version and check whether we all nodes are accessible
```

Reply:
there is a ` we` that should be removed in proposed title.